spec/models/manageiq/providers/kubevirt/infra_manager/provision/cloning_spec.rb

```diff
@@ -42,6 +47,22 @@
       expect(subject.phase_context[:new_vm_ems_ref]).to eq(new_vm.metadata.uid)
     end
 
+    context "with hash parameters" do
+      let(:options)    { {:cores_per_socket => {:cores => 2, :sockets => 2, :threads => 4}, :vm_memory => ["1024", "1024"], :root_password => "1234"} }
```

Comment:
NOTE this `:cores_per_socket => {:cores => 2, :sockets => 2, :threads => 4}` and `:spec => hash_including(:domain => {:cores => 2, :sockets => 2, :threads => 4}` tests the non-string case of the `substitute_string_param!` method https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/284/files#diff-4da493788b2d0b8f8e9969e0b7dd53c65c1c0dbe421837be10467e81a5849c13R119